pkg/blockstorage/awsebs/awsebs.go

```diff
@@ -66,21 +69,22 @@ func NewProvider(config map[string]string) (blockstorage.Provider, error) {
 }
 
 // GetConfig returns a configuration to establish AWS connection and the connected region name.
-func GetConfig(config map[string]string) (*aws.Config, string, error) {
+func GetConfig(config map[string]string) (*aws.Config, string, string, error) {
```

Comment:
It is not immediately obvious what the first and second return strings mean, even the first one alone for that matter. Given that the return types are not self documenting, let's use named return values here.
Also, please update the function comment to reflect the returned arguments.